File: Hexagono.py
from Figuras import Figuras

class Hexagono(Figuras):
    def __init__(self, lado, apotema):
        self.lado = lado
        self.apotema = apotema
    
    def area(self):
        return ((6 * self.lado) * self.apotema)/2
    
    def perimetro(self):
        return 6 * self.lado
    
    def __str__(self):
        return "Hexagono de lado: " + str(self.lado) + " y apotema: " + str(self.apotema)

File: Hexagono.java
public class Hexagono extends Figuras{
    double lado;
    double apotema;
    
    public Hexagono(double lado, double apotema){
        this.lado = lado;
        this.apotema = apotema;
    }

    @Override
    double area() {
        return ((6 * lado) * apotema)/2;
    }

    @Override
    double perimetro() {
        return 6 * lado;
    }

}
